README.md

```diff
@@ -24,34 +24,37 @@
 - `/roll d20ro+7` - Reroll only once, even if the second roll is also a 1 (lucky halflings).
 - `/roll d20r<10` - Reroll on anything less than a 10.
 - `/roll 4d6>=4` - A 4 and above is a success.
-- `/roll 4d6=5` - Can you roll exactly a 5/
+- `/roll 4d6=5` - Can you roll exactly a 5?
 - `/roll 4dF` - Fudge/fate dice.
 - `/roll (14+6)*17` - Do maths.
 - `/roll 8d8*2` - Do maths with dice.
 - `/roll floor(8d6/2)` - Devils are resistent to fire - half damage rounded down.
 
 Note that the modifiers which affect the dice rolls themselves (min, max, k etc) must come before any numerical modifiers (+4). So: `/roll 2d20kh1+9` is valid, but `/roll 2d20+9kh1` is not.
 
-The roll command has several options that you can make use of.   The first of these is the repeat option.   To use this option to roll 4d6kh3 6 times (dnd stat generation), you should do the following:
-1. type `/roll`.
-2. type in the dice you wish to roll, `4d6kh3`.
-3. Hit down arrow and click the repeat option.
-4. Type in the amount of times you wish to roll these dice, `6`.
-5. Hit enter.   The output should look as follows:
+The roll command has several options that you can make use of. The first of these is the repeat option. To use this option to roll 4d6kh3 6 times (dnd stat generation), you should do the following:
+1. Type `/roll`.
+2. Type in the dice you wish to roll: `4d6kh3`.
+3. Hit down arrow and choose the repeat option.
+4. Type in the amount of times you wish to roll these dice: `6`.
+5. Hit enter. The output should look as follows:
+```
 4d6kh3: [3, 5, 1d, 4] = 12
 4d6kh3: [1d, 4, 4, 6] = 14
 4d6kh3: [5, 4, 1d, 3] = 12
 4d6kh3: [4, 6, 3d, 5] = 15
 4d6kh3: [1d, 1, 2, 6] = 9
 4d6kh3: [5d, 5, 5, 6] = 16
+```
+The other option, output, under the roll command allows you to output information about the dice you have specified. It can output the minimum, maximum or average rolls you can get on that dice.
```

Comment:
```suggestion
The output option allows you to output information about the dice you have specified. It can output the minimum, maximum, or the average roll  that is possible with those dice.
```